prisma/seed.ts

```diff
@@ -362,16 +362,6 @@ async function seed() {
       department: "Engineering",
     },
   })
-  const rSosa = await db.profiles.upsert({
```

Comment:
I'm guessing you deleted this line because it was giving a linting error. I think we can also fix the linting error like this:

`await db.profiles.upsert({`

The error was on the unused variable.

Reply:
I deleted it because RSosa is no longer on the team, the profile is created but is not added to any project, I did not see the point of maintaining that row.